zipkin-cassandra-core/src/main/java/org/twitter/zipkin/storage/cassandra/Repository.java

```diff
@@ -205,6 +220,28 @@ public Repository(String keyspace, Cluster cluster, Boolean ensureSchema) {
                     .value("ts", QueryBuilder.bindMarker("ts"))
                     .value("trace_id", QueryBuilder.bindMarker("trace_id"))
                     .using(QueryBuilder.ttl(QueryBuilder.bindMarker("ttl_"))));
+
+        selectTraceIdsBySpanDuration = session.prepare(
+                QueryBuilder.select("d", "ts", "tid")
+                    .from("span_duration_index")
+                    .where(QueryBuilder.eq("s", QueryBuilder.bindMarker("service_name")))
```

Comment:
But what were the original length of the columns names?
What are we actually comparing here, single-char column names vs what?


bq. Only the n(name) field would have longer strings, others are usually a single number, which become comparable in space usage to column names if those are fully spelled out.

Those columns are bigints, their size are fixed.

A 5x difference doesn't sound right, given that compression will do a good job with repeated column names in such a small table.

bq. The x5 size difference may not be directly attributed to column names…

ok, this kinda worries me. How many sstables were there in each case? Was there variations in data generated too?

bq. …there are like 10 lines of code that depend on them.

That's fine if you're coming to it from within the code, but that's not always the case.

bq. But I am not strongly against changing this to long names.

What actually is a long name though? is 5 characters long? there's nothing empirical so far to go off here.

I would like want to get more empirical data down, because otherwise it smells like premature optimisation.